### DISCUSSION TITLE
Uploaded Images exist and can be downloaded, but are not visible on GitHub or in readme<title>

### DISCUSSION THREAD

**Discussion by micbay** [community/community#50450] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have an images folder in my repo for hosting the images referenced in the project readme. In my most recent upload, several images uploaded and exist in the repo to be downloaded, but they are not visible on gitHub or in readme file referencing them. These images all display fine in my local repo markdown, via VS-Code with the same relative paths. I'm not sure what could be different about these few 'invisible' images and the rest of my reference images that were created and uploaded the same way.

Here are 3 images that are not showing up:
[RacerLED-DragPreStage_800x100.png](https://github.com/micbay/RaceTimerAndController/blob/master/Images/RacerLED-DragPreStage_800x100.png)
[Adafruit_Bargraph_FullAssy_800x100.png](https://github.com/micbay/RaceTimerAndController/blob/master/Images/Adafruit_Bargraph_FullAssy_800x100.png)
[RacerLED-Idle_800x100.png](https://github.com/micbay/RaceTimerAndController/blob/master/Images/RacerLED-Idle_800x100.png)

These all become visible if I click download, but are otherwise invisible on GitHub, what might be the root cause of this?

**Comment by MohamedYansane** [comment]
Try to rename their names in your local repo on vscode then check the status and commit them all. but make sure you're working in the right branch before committing

**Comment by micbay** [comment]
Unless I'm missing something in what is a valid name, renaming, and committing didn't seem to help. I don't know if there are different varieties of .png files, but I've used the same software to create all the other images in my repo that display fine, so I'm not sure what I'd change to fix that. I will try to remake them, but it would be nice to know if this is a github bug or something in the formatting that I need to address.

Here are the renamed versions that exhibit the same issue:
[RacerLED-DragPreStage2_800x100.png](https://github.com/micbay/RaceTimerAndController/blob/master/Images/RacerLED-DragPreStage2_800x100.png)
[Adafruit_Bargraph_FullAssy2_800x100.png](https://github.com/micbay/RaceTimerAndController/blob/master/Images/Adafruit_Bargraph_FullAssy2_800x100.png)
[RacerLED-Idle2_800x100.png](https://github.com/micbay/RaceTimerAndController/blob/master/Images/RacerLED-Idle2_800x100.png)

**Reply by MohamedYansane** [reply]
look i just downloaded ur file and uploaded to my repo and it worked perfectly .
Check the file path: Ensure that the file path to the images is correct and matches the relative path used in the README file.

**Comment by micbay** [comment]
Thanks for checking the file validity of the images.
I tried the name change again. I'm positive the commits have been made, and the file paths are correct.

On further investigation, it appears to be related to my browser. I use Brave as my primary browser, but just checked the invisible files using a Chrome and Firefox browser and everything displays fine on those.

So I will have to keep digging into what might be the issue with viewing on the Brave browser. I've tried deleting cookies and restarting that browser, but there's still something preventing those few images from rendering.
if I figure out what Brave's issue is, I'll post it here.
Thanks for the help.

**Reply by MohamedYansane** [reply]
You're welcome 😊

**Comment by micbay** [comment]
Seems to be something with the Brave Browser Ad and tracking blockers; they appear to have an issue with these particular images for some reason. Github must be rendering them in such a way that causes the browser to block them.
If I change my shields to 'allow trackers and ads' then all my images show up.

It's still very strange because I can't see why those files are any different than any of the others, but at least I know the key factor involved.

**Comment by MohamedYansane** [comment]
yes you'll check it later. As you said just use another navigator until you find a solution for brave

**Comment by vemanava77** [comment]
Hey @micbay  any update on how you resolved the issue
